### PR TITLE
Read apns config via config file

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: server/configuration/base.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ database:
   port: 5432
   username: "postgres"
   password: "password"
-  database_name: "phnx_db"
+  database_name: "phnx_db"
```
Comment (raphaelrobert) [inline server/configuration/base.yaml:14]:
```suggestion
  database_name: "phnx_db"
  
```

**Review by raphaelrobert — APPROVED** [review]

File: server/configuration/base.yaml [hunk 2]
```diff
@@ -12,3 +12,4 @@ database:
   username: "postgres"
   password: "password"
   database_name: "phnx_db"
+
```
[the remark this answers is not in the document]
Comment (kkohbrok) [reply server/configuration/base.yaml]: There were whitespace on that last line. Should be fixed now.

File: server/src/endpoints/qs/push_notification_provider.rs [hunk 3]
```diff
@@ -52,41 +47,33 @@ pub struct ProductionPushNotificationProvider {
...
-                    apns_state: Some(ApnsState {
-                        key_id: config.key_id,
-                        team_id: config.team_id,
-                        private_key,
-                        token: Arc::new(Mutex::new(None)),
-                    }),
-                })
-            }
-        }
+    pub fn new(config_option: Option<ApnsSettings>) -> Result<Self, Box<dyn std::error::Error>> {
```
[the remark this answers is not in the document]
Comment (kkohbrok) [reply server/src/endpoints/qs/push_notification_provider.rs:50]: Done!

**Review by kkohbrok — COMMENTED** [review]
Thanks!

**Review by raphaelrobert — APPROVED** [review]

**Pull request merged**